src/app/content/components/Toolbar.tsx

```diff
@@ -0,0 +1,167 @@
+import React, { SFC } from 'react';
+import styled from 'styled-components';
+import {ListOl} from 'styled-icons/boxicons-regular/ListOl';
+import {Search} from 'styled-icons/fa-solid/Search';
+import {Print} from 'styled-icons/fa-solid/Print';
+import theme from '../../theme';
+
+const ListIcon = styled(ListOl)`
+  height: 2rem;
+  width: 2rem;
+  color: #027EB5;
+  margin-right: 0.7rem;
+
+  :hover {
+    color: #0064A0;
+  }
+
+`;
+
+const SearchIcon = styled(Search)`
+  height: 2rem;
+  width: 2rem;
+  color: #818181;
+  margin-right: 0.7rem;
+  position: absolute;
+
+  @media (max-width: 700px) {
+    height: ${theme.iconStyles.mobile.height};
+    width: ${theme.iconStyles.mobile.width};
+  }
+`;
+
+const PrintIcon = styled(Print)`
+  height: 2rem;
+  width: 2rem;
+  color: #818181;
+  margin-right: 0.7rem;
+
+  @media (max-width: 700px) {
+    height: ${theme.iconStyles.mobile.height};
+    width: ${theme.iconStyles.mobile.width};
+  }
+`;
+
+const TopBar = styled.div`
+  height: 5rem;
+  width: 100%;
+  max-width: 117rem;
+  font-size: 1.25rem;
+  font-weight: bold;
+  background: transparent;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  @media (max-width: 700px) {
```

Comment:
i haven't been commenting on all of them, but all these media queries need to be updated